cli/user/src/main/java/io/pravega/cli/user/stream/StreamCommand.java

```diff
@@ -65,7 +63,7 @@ public Create(@NonNull CommandArgs commandArgs) {
         public void execute() {
             ensureMinArgCount(1);
             @Cleanup
-            val sm = StreamManager.create(URI.create(getConfig().getControllerUri()));
+            val sm = StreamManager.create(getClientConfig());
```

Comment:
Please avoid using abbreviations like this one. A `streamMgr` or `streamManager`'d help reduce the cognitive burden on the person who reads the code. Same for `sc` below.

Reply:
done